CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## 8.1.1
+* New classes to install Foreman plugins:
+    * Add foreman::plugin::foreman_userdata
+    * Add foreman::plugin::foreman_snapshot_management
+* Other changes and fixes:
+    * Add retry to foreman reporting script
+    * Add the ability to get data by SSL authentication
```

Comment:
by using the proxy SSL authentication configuration?

Reply:
updated.